src/test/java/com/stripe/service/SubscriptionItemServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package com.stripe.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.google.common.collect.ImmutableMap;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.UsageRecordSummaryCollection;
+import com.stripe.net.ApiResource;
+import com.stripe.param.SubscriptionItemUsageRecordSummariesParams;
+
+import org.junit.jupiter.api.Test;
+
+class SubscriptionItemServiceTest extends BaseStripeTest {
+  private static final String ITEM_ID = "si_123";
+
+  private SubscriptionItemService subscriptionItemService = new SubscriptionItemService();
+
+  @Test
+  public void testUsageRecordSummaries() throws StripeException {
+    final UsageRecordSummaryCollection summaries = subscriptionItemService.usageRecordSummaries(
+        ITEM_ID,
+        SubscriptionItemUsageRecordSummariesParams.builder()
+            .setLimit(1L).build());
```

Comment:
This contrasts with (new) `PersonService#list` and (old) `Account#persons` where nested resource has its dedicated service.
UsageRecord, however, isn't a "core" resource, so should it need its dedicated service? 
The original way to get summaries is `SubscriptionItem#usageRecordSummaries`. 
We could implement
a) `UsageRecordSummaryService#list`  like person
b) `SubscriptionItemService#usageRecordSummaries` as done here.